frontend/src/components/Chat/ChatContainer.css

```diff
@@ -14,3 +14,13 @@ the height of the viewport */
   width: 100%;
   border-top: 1px solid #ddd;
 }
+
+.inactive {
```

Comment:
do you mind scoping this with `.chat-container .inactive`? worried about css inheritance with a common word like 'inactive'. Or we could just add specificity with `.inactive-chat-input`.